client/client-multi/client-cpp/kaa/KaaClient.hpp

```diff
@@ -102,41 +93,25 @@ class KaaClient : public IKaaClient
     IKaaClientStateStoragePtr                       status_;
     IBootstrapManagerPtr                            bootstrapManager_;
     std::unique_ptr<ProfileManager>                 profileManager_;
-#ifdef KAA_USE_NOTIFICATIONS
     std::unique_ptr<NotificationManager>            notificationManager_;
-#endif
 
     std::unique_ptr<KeyPair> clientKeys_;
-    std::string     publicKeyHash_;
+    std::string              publicKeyHash_;
 
-#ifdef KAA_USE_CONFIGURATION
     std::unique_ptr<ConfigurationManager>            configurationManager_;
     std::unique_ptr<ConfigurationProcessor>          configurationProcessor_;
     std::unique_ptr<ConfigurationPersistenceManager> configurationPersistenceManager_;
-#endif
-#ifdef KAA_USE_EVENTS
-    std::unique_ptr<EventManager>         eventManager_;
-    std::unique_ptr<EventFamilyFactory>   eventFamilyFactory_;
-    std::unique_ptr<EndpointRegistrationManager>    registrationManager_;
-#endif
-    std::unique_ptr<IKaaChannelManager>   channelManager_;
-    std::unique_ptr<SyncDataProcessor>  syncProcessor_;
-
-#ifdef KAA_DEFAULT_BOOTSTRAP_HTTP_CHANNEL
+    std::unique_ptr<EventManager>                    eventManager_;
+    std::unique_ptr<EventFamilyFactory>              eventFamilyFactory_;
+    std::unique_ptr<EndpointRegistrationManager>     registrationManager_;
+    std::unique_ptr<LogCollector>                    logCollector_;
+    std::unique_ptr<IKaaChannelManager>              channelManager_;
+    std::unique_ptr<SyncDataProcessor>               syncProcessor_;
+
     std::unique_ptr<DefaultBootstrapChannel>          bootstrapChannel_;
```

Comment:
the declaration of the variable 'bootstrapChannel_' is a little bit shifted to the right. e.g code style